### ISSUE TITLE
Refactor client commands as loom NOUN VERB

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#437]
For consistency, use loom, then the object type being manipulated, then that action performed, then arguments for that action. This gives a lot of flexibility and keeps parser options for each verb separate. For example:

    loom run create TEMPLATE [INPUT, [INPUT, ...]]
    loom run kill RUNID
    loom run pause RUNID
    loom run delete RUNID

But using "loom run TEMPLATE [INPUT, [INPUT, ...]]" instead, it hard to implement new functions without having ambiguity in parsing the arguments.

This is coming up for "TAG" as well. "

    loom tag TARGET TAGNAME 

is simple, but then to delete a tag we have to

    loom tag -d TAG 

and check to ensure that positional TARGET and TAG arguments are omitted. To view tags we have to look somewhere else, under 

    loom show tag

which reduces discoverability. The commands should be something like:

    loom tag create
    loom tag delete
    loom tag show [TAGNAME]

**Comment by nhammond** [comment]
Current:

```
loom -h
usage: loom [-h] [-v] {run,server,import,export,show,tag,browser,test} ...

positional arguments:
  {run,server,import,export,show,tag,browser,test}
                        select a subcommand
    run                 run a template
    server              manage the Loom server
    import              import files or other data to the Loom server
    export              export files or other data from the Loom server
    show                query and show data objects from the Loom server
    tag                 manage tags
    browser             launch the Loom web browser
    test                run tests

optional arguments:
  -h, --help            show this help message and exit
  -v, --version
```

proposed:

```
loom -h
usage: loom [-h] [-v] {run,template,file,test,tag,label,browser,server,test}
```

**Comment by nhammond** [comment]
import, export, and show can be kept with a deprecation warning but hidden

**Comment by nhammond** [comment]
updated proposal:

**Comment by nhammond** [comment]
updated proposal:

```
loom server start
loom server stop
loom server delete

loom browser

loom template import
loom template export
loom template delete

loom file import
loom file export
loom file delete

loom run start
loom run stop
loom run delete

# Repeat “start”-->”end” for run, file, and template. Shown with run.
# start
loom run tag create TARGET TAG
loom run tag delete TAG
loom run tag list TAG # show this tag
loom run tag list # show all tags

loom run label create TARGET LABEL
loom run label delete LABEL
loom run label list LABEL # show this label
loom run label list # show all labels
# end

loom file import SOURCE --tag x --label y --label z
loom template import SOURCE --tag x --label y --label z
loom run start TEMPLATE [INPUT1 …] --tag x --label y --label z
```

**Comment by nhammond** [comment]
Fixed in release 0.5.0
